### DISCUSSION TITLE
Here's how I manage to install the version ^2.0.0 on windows.

### DISCUSSION THREAD

**Discussion by endyjasmi** [josephg/node-foundationdb#73] [General]
When I try to install in on windows, it complained about this
`C:\Program Files\foundationdb\include\foundationdb\fdb_c.h(61,10): fatal  error C1083: Cannot open include file: 'fdb_c_types.h': No such file or directory`.

All I did was copy from [https://github.com/apple/foundationdb/blob/main/bindings/c/foundationdb/fdb_c_types.h](https://github.com/apple/foundationdb/blob/main/bindings/c/foundationdb/fdb_c_types.h) and place it at `C:\Program Files\foundationdb\include\foundationdb\fdb_c_types.h`. 

And it install fine. 😁

I do know that windows is sort of second class citizen with foundationdb, but hey, it works for now. cheers

**Comment by josephg** [comment]
Thanks! Do you want to put together a PR for the github action script?

**Reply by endyjasmi** [reply]
I don't mine a PR but I'm not sure anything to do with the package as it seems to work fine as long as the right header file is in place.

As for PR for FoundationDB, there seems to already includes the `fdb_c_types.h` in the installer [here](https://github.com/apple/foundationdb/blob/main/packaging/msi/FDBInstaller.wxs#L107).

Let me know what can I do.
